marketplace-and-integrations/integrations/algolia.md

```diff
@@ -113,16 +113,74 @@ The list of default properties consists of:
 
 Indexing the content for Algolia is based on the `IDataEditor.PropertyIndexValueFactory` property from Umbraco CMS, the indexed value of the property being retrieved using the `GetIndexValues` method.
 
+The integration uses the same conversion process as Umbraco CMS uses for Examine, and apply a custom converter afterwards.
+
 The [`ContentBuilder`](https://github.com/umbraco/Umbraco.Cms.Integrations/blob/main/src/Umbraco.Cms.Integrations.Search.Algolia/Builders/ContentRecordBuilder.cs) is responsible for creating the record object that will be pushed to _Algolia_ and the [`AlgoliaSearchPropertyIndexValueFactory`](https://github.com/umbraco/Umbraco.Cms.Integrations/blob/main/src/Umbraco.Cms.Integrations.Search.Algolia/Services/AlgoliaSearchPropertyIndexValueFactory.cs) implementation of `IAlgoliaSearchPropertyIndexValueFactory` will return the property value.
 
-The current implementation contains a custom converter for the `Umbraco.MediaPicker3` property editor.
+To customize the returned value from Umbraco CMS you would need to use a custom converter specific to the particular indexed Umbraco property editor.
+
+To extend the behavior, there are available options:
+
+1. Version 2.0.0 and up 
```

Comment:
Instead of adding the 2 options as numbers, they can be added as headings. Then the steps for each options can be setup with steps (1, 2, 3 etc instead of a, b, c etc)
```suggestion
### Version 2.0.0 and up 
```